Environment/Image/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM mcr.microsoft.com/devcontainers/python:3.11
```

Comment:
Let's rename `Environment` to `environment`.  Let's stick with all-small singular name for directories.

Similar to `swe-rex` folder, please create one folder named `local_docker` and move the `LocalDockerEnvironment.py` file inside. If I'm not wrong, other `Dockerfile` and related scripts are useful only to build the image. They all can be grouped under `local_docker/image_builder` folder.

When you're renaming, please make sure to update the imports from `agent.py` also.